mautic/mautic
# Add gated video css to iframeStyle

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | n
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Styling for the gated video on the content tab quit working in staging recently. This fixes it.


#### Steps to test this PR:
1. Apply the PR
2. Ensure gated video styling looks good on landing page edit outside of the builder.
